streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -115,7 +115,7 @@ public void suspend() {
     }
 
     @Override
-    public void resume() {
+    public void resume(final boolean requiresUpdate) {
```

Comment:
Why do we use third-person singular instead of calling the flag `requireUpdate` or `updateNeeded`?

Reply:
agree. beautified it.